src/main/webapp/i18n/de/faq.json

```diff
@@ -8,7 +8,8 @@
                 "accept": "FAQ akzeptieren",
                 "reject": "FAQ ablehnen",
                 "filterLabel": "Filter",
-                "createOrEditLabel": "FAQ erstellen oder bearbeiten"
+                "createOrEditLabel": "FAQ erstellen oder bearbeiten",
+                "ingestLabel": "FAQs an IRIS schicken"
```

Comment:
If a user does not know too much about IRIS, this might not be too descriptive IMO. What benefits would I have of doing that? e.g. sth like "IRIS/chatbot will use the FAQs to answer questions more precisely..."

Reply:
I wanted to stay consistent with the button in the lecture UI. I am gratefull for any short suggestions for the button, since i also think this is not a good solution, but i cant explain it in detail either